WordPress/src/main/java/org/wordpress/android/ui/prefs/AppPrefs.java

```diff
@@ -624,14 +625,14 @@ public static void removeAppWideEditorPreference() {
         remove(DeletablePrefKey.GUTENBERG_DEFAULT_FOR_NEW_POSTS);
     }
 
-    public static boolean shouldShowGutenbergInfoPopupForTheNewPosts(String siteURL) {
+    public static boolean getShowGutenbergInfoPopupForTheNewPosts(PrefKey key, String siteURL) {
```

Comment:
I wonder, does the `setShowGutenbergInfoPopupForTheNewPosts()` call in line 646 below need to adopt a "PrefKey" param as well? I'm a bit confused by the codepaths but, it looks to me that when the flag is true, we'll try to clear the Phase 1 flag only.

Reply:
Updated in 47d5282d7f